conf.d/docker_daemon.yaml.example

```diff
@@ -51,6 +51,12 @@ instances:
     #       enable if absolutely necessary.
     # custom_cgroups: false
 
+    # Report docker container healthcheck events as service checks
+    # Note: enabling this option modifies the way in which we inspect the containers and causes
+    #       some overhead - if you run a high volume of containers we may timeout. Please only
+    #       enable if absolutely necessary.
```

Comment:
I think you can remove the last sentence. The slow part of inspecting a container is the lock acquisition over said container, and this is also done in `docker ps`, so we may timeout the check but it could also happen without the inspect part.
Same for the `custom_cgroups` option actually, the last sentence should be dropped.